pkg/app/apps_reconciler_test.go

```diff
@@ -1,14 +1,14 @@
 // Copyright 2021 VMware, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-package controller_test
+package app_test
 
 import (
 	"reflect"
 	"testing"
 
-	"github.com/vmware-tanzu/carvel-kapp-controller/cmd/controller"
 	"github.com/vmware-tanzu/carvel-kapp-controller/pkg/apis/kappctrl/v1alpha1"
+	app_ "github.com/vmware-tanzu/carvel-kapp-controller/pkg/app"
```

Comment:
Can we use something besides `_` here? Maybe even just call appDep or appPkg?

Reply:
i think typically camelcase is not used in aliases. apppkg?

Reply:
renamed to apppkg